src/gui/editors/SongEditor.cpp

```diff
@@ -1029,6 +1028,17 @@ SongEditorWindow::SongEditorWindow(Song* song) :
 
 	connect(song, SIGNAL(projectLoaded()), this, SLOT(adjustUiAfterProjectLoad()));
 	connect(this, SIGNAL(resized()), m_editor, SLOT(updatePositionLine()));
+
+	// In case our current audio device does not support capture,
+	// disable the record buttons.
+	if (!Engine::audioEngine()->captureDeviceAvailable())
+	{
+		for (auto &recordAction : {m_recordAccompanyAction, m_recordAction})
```

Comment:
I was able to test with the "m_recordAccompanyAction" (I guess that's the record button with the play sign), but when I pressed `m_recordAction` (I guess that's the record button without play sign), just nothing happened - the cursor did not move and nothing was recorded.

Do you know why? What steps are required to make any use of `m_recordAction`?

Reply:
Oh, this is also mentioned in #5990 .